robotoff/insights/question.py

```diff
@@ -141,16 +148,13 @@ def format_question(self, insight: ProductInsight, lang: str) -> Question:
         )
 
     @staticmethod
-    def get_source_image_url(barcode: str) -> Optional[str]:
-        product: Optional[JSONType] = get_product(barcode, fields=["selected_images"])
+    def get_source_image_url(self, barcode: str) -> Optional[str]:
```

Comment:
`get_source_image_url` is declared as static method so you cannot add a self parameter.